### PR TITLE
Fixing the Extras directory

### PR DESCRIPTION
**Pull request by ledm**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [ ] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [ ] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [ ] Give this pull request a descriptive title.
- [ ] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [ ] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes {#47}

### CODE REVIEW

File: _extras/about.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 ---
 title: About
 ---
+
+## ESMValTool 
+
+The Earth System Model Evaluation Tool (ESMValTool) is a community-development that aims at improving diagnosing and understanding of the causes and effects of model biases and inter-model spread. The ESMValTool is open to both users and developers encouraging open exchange of diagnostic source code and evaluation results from the Coupled Model Intercomparison Project (CMIP) ensemble. This will facilitate and improve ESM evaluation beyond the state-of-the-art and aims at supporting the activities within CMIP and at individual modelling centers. We envisage running the ESMValTool routinely on the CMIP model output utilizing observations available through the Earth System Grid Federation (ESGF) in standard formats (obs4MIPs) or made available at ESGF nodes.
```
Comment (hb326) [inline _extras/about.md:7]: Should that be "community development" instead of "community-development"?

Comment (hb326) [inline _extras/about.md:7]: Maybe remove the "of" in "... understanding of the causes and effects...".

File: _extras/about.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+
+The goal is to develop a benchmarking and evaluation tool that produces well-established analyses as soon as model output from CMIP simulations becomes available, e.g., at one of the central repositories of the ESGF. This is realized through standard recipes that reproduce a certain set of diagnostics and performance metrics that have demonstrated its importance in benchmarking Earth System Models (ESMs) in a paper or assessment report, such as Chapter 9 of the Intergovernmental Panel on Climate Change (IPCC) Fifth Assessment Report (AR5) (Flato et al., 2013). The expectation is that in this way a routine and systematic evaluation of model results can be made more efficient, thereby enabling scientists to focus on developing more innovative methods of analysis rather than constantly having to “reinvent the wheel”.
+
+## ESMValTool community.
+
+The ESMValTool community includes scientists and programmers from around the world.
```
Comment (hb326) [inline _extras/about.md:13]: Do we want to mention numbers here? Would be nice since it shows that it is indeed a community tool, but then we might have to update these numbers on a regular basis...

File: _extras/about.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
+
+The goal is to develop a benchmarking and evaluation tool that produces well-established analyses as soon as model output from CMIP simulations becomes available, e.g., at one of the central repositories of the ESGF. This is realized through standard recipes that reproduce a certain set of diagnostics and performance metrics that have demonstrated its importance in benchmarking Earth System Models (ESMs) in a paper or assessment report, such as Chapter 9 of the Intergovernmental Panel on Climate Change (IPCC) Fifth Assessment Report (AR5) (Flato et al., 2013). The expectation is that in this way a routine and systematic evaluation of model results can be made more efficient, thereby enabling scientists to focus on developing more innovative methods of analysis rather than constantly having to “reinvent the wheel”.
+
+## ESMValTool community.
+
+The ESMValTool community includes scientists and programmers from around the world. 
+
+ESMValTool has been used to analyse data
```
Comment (hb326) [inline _extras/about.md:15]: I think there is some part of the sentence/paragraph missing.

File: _extras/about.md [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
+## ESMValTool community.
+
+The ESMValTool community includes scientists and programmers from around the world. 
+
+ESMValTool has been used to analyse data 
+
+
+## OBS4Mips
+
+In parallel to standardization of model output, the ESGF also hosts observations for Model Intercomparison Projects (obs4MIPs) and reanalyses data (ana4MIPs). obs4MIPs provides open access data sets of satellite data that are comparable in terms of variables, temporal and spatial frequency, and periods to CMIP model output (Taylor et al., 2012). The ESMValTool utilizes these observations and reanalyses from ana4MIPs plus additionally available observations in order to evaluate the models performance. In many diagnostics and metrics, more than one observational data set or meteorological reanalysis is used to assess uncertainties in observations.
```
Comment (hb326) [inline _extras/about.md:20]: Do you mean in the last sentence "... is used to account for uncertainties in observations."?

File: _extras/about.md [hunk 5]
```diff
@@ -1,5 +1,41 @@
...
+
+The main idea of the ESMValTool is to provide a broad suite of diagnostics which can be performed easily when new model simulations are run. The suite of diagnostics needs to be broad enough to reflect the diversity and complexity of Earth System Models, but must also be robust enough to be run routinely or semi-operationally. In order the address these challenging objectives the ESMValTool is conceived as a framework which allows community contributions to be bound into a coherent framework.
```
Comment (hb326) [inline _extras/about.md:22]: Maybe rephrase the first sentence to "... when new model simulations become available."?

Comment (hb326) [inline _extras/about.md:22]: We have already introduced an abbreviation for "Earth System Models" and could use it here.

Comment (hb326) [inline _extras/about.md:22]: Rephrase the last sentence to "In order to address these..."

File: _extras/about.md [hunk 6]
```diff
@@ -1,5 +1,41 @@
...
+
+## License
+The ESMValTool is released under the Apache License, version 2.0. Citation of the ESMValTool paper (“Software Documentation Paper”) is kindly requested upon use, alongside with the software DOI for ESMValTool (doi:10.5281/zenodo.3401363) and ESMValCore (doi:10.5281/zenodo.3387139) and version number:
```
Comment (hb326) [inline _extras/about.md:25]: Why did you put the Software Documentation Paper in " "? Because it is not actually called this?

Comment (hb326) [inline _extras/about.md:25]: The version number is still missing here. Do we want to have that included in a way that it is updated automatically when necessary?

File: _extras/about.md [hunk 7]
```diff
@@ -1,5 +1,41 @@
...
...n 2020 Framework Programme for Research and Innovation “Infrastructure for the European Network for Earth System Modelling (IS-ENES3)” project under grant agreement no. 824084; (3) the European Union's Horizon 2020 Framework Programme for Research and Innovation “Coordinated Research in Earth Systems and Climate: Experiments, kNowledge, Dissemination and Outreach (CRESCENDO)” project under grant agreement no. 641816; (4) the the European Union's Horizon 2020 Framework Programme for Research and Innovation “PRocess-based climate sIMulation: AdVances in high-resolution modelling and European climate Risk Assessment (PRIMAVERA)” project under grant agreement no. 641727; (5) the Helmholtz Society project “Advanced Earth System Model Evaluation for CMIP (EVal4CMIP)”; (6) project S1 (Diagnosis and Metrics in Climate Models) of the Collaborative Research Centre TRR 181 “Energy Transfer in Atmosphere and Ocean” funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) project no. 274762653; and (7) National Environmental Research Council (NERC) National Capability Science Multi-Centre (NCSMC) funding for the UK, Earth System Modelling project (grant no. NE/N018036/1).
```
Comment (hb326) [inline _extras/about.md:37]: I think "Eval4CMIP" needs a small letter "v" in it.

File: _extras/about.md (hunk 6 above)
Comment (ledm) [reply _extras/about.md:25]: Lol, I copied and pasted this entire section directly from the Readthedocs page.

File: _extras/about.md (hunk 7 above)
Comment (ledm) [reply _extras/about.md:37]: Also, I copied and pasted this directly from the publication!

**Comment by ledm** [comment]
I think the extras folder would be a good place to list all the people who have worked on the tutorial in the user engagement group. Are we happy to do that or should we just say the "user engagement group"?

**Comment by ledm** [comment]
I think there was too much text in the about.md page, so I re-wrote it to be clear, simple yet cover enough. (Haven't been able to do a local build to test it though)

File: _extras/about.md [hunk 8]
```diff
@@ -1,5 +1,98 @@
...
+which is of vital importance for securing future funding.
+
+You should consider this an obligation if you have taken advantage of the
+ESMValTool, which represents the end product of considerable effort by the
+development teams.
+
+
+## Acknowledgements
+
+The technical development work for ESValTool v2.0 was funded by various projects,
```
Comment (ledm) [inline _extras/about.md:76]:
```suggestion
The technical development work for ESMValTool v2.0 was funded by various projects,
```

File: _extras/about.md [hunk 9]
```diff
@@ -1,5 +1,98 @@
...
+“Infrastructure for the European Network for Earth System Modelling (IS-ENES3)”
+project under grant agreement no. 824084; (3) the European Union's Horizon 2020
+Framework Programme for Research and Innovation “Coordinated Research in Earth
+Systems and Climate: Experiments, kNowledge, Dissemination and Outreach (CRESCENDO)”
+project under grant agreement no. 641816; (4) the the European Union's Horizon
+2020 Framework Programme for Research and Innovation “PRocess-based climate
+sIMulation: AdVances in high-resolution modelling and European climate Risk
+Assessment (PRIMAVERA)” project under grant agreement no. 641727; (5) the
+Helmholtz Society project “Advanced Earth System Model Evaluation for CMIP
+(EVal4CMIP)”; (6) project S1 (Diagnosis and Metrics in Climate Models) of
```
Comment (ledm) [inline _extras/about.md:89]:
```suggestion
(Eval4CMIP)”; (6) project S1 (Diagnosis and Metrics in Climate Models) of
```

File: _extras/guide.md [hunk 10]
```diff
@@ -1,6 +1,78 @@
 ---
 title: "Instructor Notes"
 ---
-FIXME
+
+This page includes some tips, reminders and advice for giving this tutorial.
```
Comment (ledm) [inline _extras/guide.md:5]: Probably need a blank line after this?

File: _extras/guide.md [hunk 11]
```diff
@@ -1,6 +1,78 @@
...
+ - Venue details (if on site) or meeting connection details (if online).   
+
+Also allow for sufficient time to get access to any 
+accounts etc. which could take more than a few days in some cases (a good 
+thumb rule is to have the time to send a reminder email a few days after
+the first email, 
+so that participants will still have time after the reminder to complete 
+setup). 
+
+Plan for the tutorial to last at least two hours (this could vary based on
```
Comment (ledm) [inline _extras/guide.md:42]: We know that our tutorial takes two hours, but does this tutorial take two hours as well?
